src/emulator/functionsEmulatorRuntime.ts

```diff
@@ -312,16 +315,7 @@ function InitializeNetworkFiltering(frb: FunctionsRuntimeBundle): void {
       try {
```

Comment:
This `try` and `catch` are the same ... that can't be right.

Reply:
False. One is `return original()` and one is `return new original()`